### ISSUE TITLE
[Bug][a11y]: Picker button is not using high contrast colors when focused or disabled

### ISSUE THREAD

**Issue by dineshy87** [adobe/spectrum-web-components#3402]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Picker Button

### Expected behavior

[Picker button](https://opensource.adobe.com/spectrum-web-components/components/picker-button/) must use high contrast system colors when using forced-colors:active

### Actual behavior

Looks like it is still picking colors from non high contrast mode

**Comment by najikahalsema** [comment]
We should check to see if this is still true post-Core Tokens Migration.

**Comment by blaabaer** [comment]
It appears that on the SWC dev site that the Picker visuals match the ActionButton.

Testing via:
- "Increase contrast" turned on in Mac system accessbility settings
- Using Safari

<img width="1436" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/7ba93681-a6b5-4765-ad3f-133322537798">
<img width="1436" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/065e87e0-eaa3-4056-a7d8-f8dd29b7cb81">
<img width="1436" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/8bc063b2-f3c8-4d46-be69-6e6e6dbdb9fe">

**Comment by Westbrook** [comment]
Thanks for starting to dig into this @blaabaer!

Be sure that you looking at [Picker Button](https://opensource.adobe.com/spectrum-web-components/components/picker-button/) which is a different pattern than what you've taking screenshots of so far. This element is leveraged within the [Combobox](https://opensource.adobe.com/spectrum-web-components/components/combobox/) element, as well.

**Comment by blaabaer** [comment]
Combobox and PickerButton components are also significatly different (high contrast on the left, normal on the right)

<img width="2300" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/44edd6cc-f471-48f3-91e0-56ce91704a97">
<img width="2306" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/054ef516-3e3e-47b0-95c7-72d1394818bc">
<img width="2295" alt="image" src="https://github.com/adobe/spectrum-web-components/assets/162190334/46d47cf2-d536-4669-90d6-86d5a9176691">

**Comment by Westbrook** [comment]
Can't be sure, but your left vs right comparison looks like "Spectrum" vs "Express" at the theme level. Be sure you're using forced color emulation which can be managed [like this](https://developer.chrome.com/docs/devtools/rendering/emulate-css#emulate_css_media_feature_forced-colors) in Chromium.

**Comment by blaabaer** [comment]
You're right, and that was my mistake. For some reason, when I flipped "High Contrast" in system settings, my browser content actually barely changed. "Forced Color Emulation" like you described changes the visuals significantly:

<details>
<summary> Action Button Forced-Color ON vs OFF, Spectrum and Express theme</summary>

![image](https://github.com/adobe/spectrum-web-components/assets/162190334/37f8da20-274e-453d-bf14-f07b96c8e3d3)
![image](https://github.com/adobe/spectrum-web-components/assets/162190334/624d6871-7621-4a43-8e29-fbb600427c1b)

</details>

<details>
<summary> Picker Button Forced-Color ON vs OFF, Spectrum and Express theme</summary>

(focused on first dropdown button)
![image](https://github.com/adobe/spectrum-web-components/assets/162190334/97fc7751-6364-49b1-aaf8-550a1ef7a00b)

(focused on first dropdown button)
![image](https://github.com/adobe/spectrum-web-components/assets/162190334/7c0d3cd9-af5b-4a75-b90e-9a2e3b2c550e)

</details>

<details>
<summary> Combobox Forced-Color ON vs OFF, Spectrum and Express theme</summary>

![image](https://github.com/adobe/spectrum-web-components/assets/162190334/440b0f96-3912-4c63-83cf-ccaa88ab1a49)

![image](https://github.com/adobe/spectrum-web-components/assets/162190334/a8c1d268-7a9a-43ab-b5a2-5f031d30c475)

</details>

The ticket description specifically points out focused and disabled. Focused states for all these components in Forced-Color now have a black outline that is unique to forced-color and different from the regular colors, so I think that's no longer an issue.

The Disabled/Invalid states in Forced-Color appear identical to normal buttons. I'm not sure whether that is considered standard, or a problem. I also noticed that the description for the "Invalid" Picker Button case describes a red border that doesn't actually exist.

![image](https://github.com/adobe/spectrum-web-components/assets/162190334/a8114b5e-2a6a-4bf8-b84d-7c6090d88379)

**Comment by najikahalsema** [comment]
SWC-278

**Comment by najikahalsema** [comment]
This will be fixed as part of the migration to Spectrum 2. Design is currently in the process of addressing this in their design specs.
